docs/styles/spec/file-input-spec.less

```diff
@@ -1,4 +1,4 @@
-@import (reference) 'components/button/index';
+//@import (reference) 'components/button/index';
```

Comment:
Put the LESS file back with minimum CSS so that we don't break this file.

Reply:
Better yet, migrate _just_ the stuff relating to the buttons into SCSS (split spec styles among both LESS and SCSS).